charts/hivemq-swarm/values.yaml

```diff
@@ -0,0 +1,240 @@
+image:
+  repository: hivemq/hivemq-swarm
+  pullPolicy: IfNotPresent
+  tag: "4.6.0-RC2"
+imagePullSecrets: []
+
+## Cluster domain
+##
+clusterDomain: cluster.local
+
+## HiveMQ Swarm license as base64 encoded string.
+## NOTE: it's also possible to mount a license via a secret/configmap, please see the commander extraVolumes and extraVolumeMounts section
+##
+license: ""
+
+## HiveMQ Swarm commander configuration
+##
+commander:
+  logLevel: info
+
+  ## Configure resources for HiveMQ Swarm Commander Pod (Note: request = limits for HiveMQ Swarm)
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    cpu: 500m
+    memory: 1G
+
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ##
+  podSecurityContext: { }
+
+  ## Configure HiveMQ Swarm Commander Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ##
+  securityContext: {}
+
+  ## Configure affinity preset for HiveMQ Swarm Commander Pod
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node
+  ##
+  affinity: {}
+    # podAntiAffinity:
+    #   preferredDuringSchedulingIgnoredDuringExecution:
+    #     # Try not to co-locate with other swarm instances
+    #     - weight: 50
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: app.kubernetes.io/instance
+    #               operator: In
+    #               values:
+    #                 - ${LABEL}
+    #         topologyKey: "kubernetes.io/hostname"
+    #     # Try not to co-locate with HiveMQ nodes
+    #     - weight: 60
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: app
+    #               operator: In
+    #               values:
+    #                 - hivemq
+    #         topologyKey: "kubernetes.io/hostname"
```

Comment:
Right now the example wasn't tested - not sure if it works. Maybe we drop this in the first release or update it with a working example?

Reply:
Yes, I think it's ok to just drop the example.